userdata/MaintainSystem.ps1

```diff
@@ -65,9 +65,13 @@ function Remove-OldTaskDirectories {
   process {
     foreach ($target in ($targets | ? { (Test-Path -Path ('{0}:\' -f $_[0]) -ErrorAction SilentlyContinue) })) {
       $all_task_paths = @(Get-ChildItem -Path $target | Sort-Object -Property { $_.LastWriteTime })
```

Comment:
On reflection, I wonder if we should sort by creation time, rather than last write time, in case generic-worker touches other directories when it tries to purge them and isn't successful.